crates/ruma-common/src/events/audio.rs

```diff
@@ -52,6 +56,31 @@ impl AudioEventContent {
     pub fn with_message(message: MessageContent, file: FileContent) -> Self {
         Self { message, file, audio: Default::default(), relates_to: None }
     }
+
+    /// Create a new `AudioEventContent` from the given `AudioMessageEventContent` and optional
+    /// relation.
+    pub fn from_audio_room_message(
+        content: AudioMessageEventContent,
+        relates_to: Option<Relation>,
+    ) -> Self {
+        let AudioMessageEventContent {
+            body,
+            src,
+            info,
+            message,
+            file,
+            audio,
+            #[cfg(feature = "unstable-msc3245")]
+                voice: _,
```

Comment:
```suggestion
            voice: _,
```

Reply:
Sounds like a bug, you can add `#[rustfmt::skip]` to the overall statement to stop it from touching that.